### ISSUE TITLE
🟡 Activations README strategy table omits SOFTMAX — 38 of 39 registered squashes documented

### ISSUE THREAD

**Issue by stservice** [stSoftwareAU/NEAT-TS#3690]
`src/methods/activations/README.md:58-97` — the backpropagation-strategy table covers 38 activation functions, but the central registry (`src/methods/activations/Activations.ts:201-256`) registers 39. The missing entry is `SOFTMAX` (imported at `Activations.ts:44` from `src/methods/activations/types/SOFTMAX.ts`): it appears nowhere in the README — no priority weight, no invertibility note, no derivative-vs-unSquash recommendation. `CONTRIBUTING.md:471` calls this README "the full reference" for activation functions, and `CONTRIBUTING.md:498-501` requires every new activation to add an entry "following the existing format — include priority, invertibility, and backpropagation strategy" (checks 2 and 6, `severity:medium`).

## Why this matters

The table presents itself as the complete per-squash strategy reference, so a reader reasonably concludes `SOFTMAX` does not exist as a squash — despite it being registered for mutation selection and load-bearing in practice (it is the output activation the `CROSS_ENTROPY` cost coupling selects, used by downstream factory-seeded classifiers such as the MNIST example in NEAT-AI-Examples). A contributor auditing priorities or backprop strategies from this table silently skips one real function, and the documented add-an-entry rule has already been violated once, which invites further drift.

## Suggested fix

Add a `SOFTMAX` row to the table in `src/methods/activations/README.md` following the existing format — its priority weight, invertibility (softmax is not element-wise invertible), and the recommended error-propagation strategy (derivative-based via the softmax Jacobian / cross-entropy shortcut). Consider also adding a one-line note that the registry in `Activations.ts` is the authoritative list, so the next mismatch is caught at review time.

**Comment by VibeCoderST** [comment]
✅ **Vibe Coder released this claim** — host `GRQ-23`, finished 16:58 UTC.

**Comment by VibeCoderST** [comment]
## Automated Processing Failed (First Attempt)

**Category:** `quality-failure`

The automated worker was unable to complete this issue on the first attempt.

### Error Output
> workflow hygiene: FAILED
> workflow hygiene: FAILED
> workflow hygiene: FAILED

<details>
<summary>Full output (click to expand)</summary>

Quality checks (`./quality.sh`) failed and could not be fixed automatically.

The PR was not created because local quality checks must pass before raising a PR.

### Quality Check Output
```
=== Running Quality Checks ===
deno test directory not found
deno source directories not found
deno source directories not found
deno source directories not found
workflow hygiene: FAILED
VIOLATION: .github/workflows/github-release.yml:34: multi-line `run:` block does not open with `set -euo pipefail`
VIOLATION: .github/workflows/publish.yml:59: multi-line `run:` block does not open with `set -euo pipefail`
VIOLATION: .github/workflows/publish.yml:179: multi-line `run:` block does not open with `set -euo pipefail`

Workflow hygiene defects detected (Issue #3716). Every multi-line
`run:` block must open with `set -euo pipefail`, and one pinned SHA
must carry one version comment everywhere it appears.
config integration: SKIPPED (deno or .config.json not available)

Running independent checks in parallel (Issue #625)...
source target validation: PASSED (0 targets checked)
pages-liquid: SKIPPED (Ruby + Liquid toolchain not available (run `bundle install` to enable))
mermaid: PASSED (351 file(s), 347 block(s) checked)
markdownlint: PASSED (351 file(s) checked)
docs prompt versions: SKIPPED (no prompts/ directory)
```

<details>
<summary>Full quality check output (click to expand)</summary>

```
=== Running Quality Checks ===
deno test directory not found
deno source directories not found
deno source directories not found
deno source directories not found
workflow hygiene: FAILED
VIOLATION: .github/workflows/github-release.yml:34: multi-line `run:` block does not open with `set -euo pipefail`
VIOLATION: .github/workflows/publish.yml:59: multi-line `run:` block does not open with `set -euo pipefail`
VIOLATION: .github/workflows/publish.yml:179: multi-line `run:` block does not open with `set -euo pipefail`

Workflow hygiene defects detected (Issue #3716). Every multi-line
`run:` block must open with `set -euo pipefail`, and one pinned SHA
must carry one version comment everywhere it appears.
config integration: SKIPPED (deno or .config.json not available)

Running independent checks in parallel (Issue #625)...
source target validation: PASSED (0 targets checked)
pages-liquid: SKIPPED (Ruby + Liquid toolchain not available (run `bundle install` to enable))
mermaid: PASSED (351 file(s), 347 block(s) checked)
markdownlint: PASSED (351 file(s) checked)
docs prompt versions: SKIPPED (no prompts/ directory)
```

</details>

The issue will be retried on the next scan. Claude will attempt to fix the quality issues.

> **Note:** Quality checks were already failing on the clean repository before the worker started. The following pre-existing failures were detected:
>
```
=== Running Quality Checks ===
deno test directory not found
deno source directories not found
deno source directories not found
deno source directories not found
workflow hygiene: FAILED
VIOLATION: .github/workflows/github-release.yml:34: multi-line `run:` block does not open with `set -euo pipefail`
VIOLATION: .github/workflows/publish.yml:59: multi-line `run:` block does not open with `set -euo pipefail`
VIOLATION: .github/workflows/publish.yml:179: multi-line `run:` block does not open with `set -euo pipefail`

Workflow hygiene defects detected (Issue #3716). Every multi-line
`run:` block must open with `set -euo pipefail`, and one pinned SHA
must carry one version comment everywhere it appears.
config integration: SKIPPED (deno or .config.json not available)

Running independent checks in parallel (Issue #625)...
source target validation: PASSED (0 targets checked)
pages-liquid: SKIPPED (Ruby + Liquid toolchain not available (run `bundle install` to enable))
mermaid: PASSED (350 file(s), 346 block(s) checked)
markdownlint: PASSED (350 file(s) checked)
docs prompt versions: SKIPPED (no prompts/ directory)
```
</details>

### What this means
Likely cause: changes failed quality checks.

### What happens next?
- This issue has been marked with the `failed-once` label
- The worker will **automatically retry** this issue on the next scan
- If it fails again, the issue will be marked as `failed` and require manual intervention

### To prevent automatic retry
If you want to work on this manually, remove the `failed-once` label and assign yourself.

**Comment by VibeCoderST** [comment]
Closed automatically — PR has been merged.
